src/test/java/uk/nhs/hee/trainee/details/service/ProgrammeMembershipServiceTest.java

```diff
@@ -398,6 +472,20 @@ void shouldCacheCojFromDeleteProgrammeMembershipsWhenCojSignedForMultipleCurricu
     verify(cachingDelegate, times(1)).cacheConditionsOfJoining(eq("789"), any());
   }
 
+  @Test
+  void shouldCacheCojFromDeleteProgrammeMembershipsWhenCojSignedUsingUuid() {
+    TraineeProfile traineeProfile = new TraineeProfile();
+    traineeProfile.getProgrammeMemberships()
+        .add(createProgrammeMembership(PROGRAMME_MEMBERSHIP_UUID.toString(), ORIGINAL_SUFFIX, 0));
+
+    when(repository.findByTraineeTisId(TRAINEE_TIS_ID)).thenReturn(traineeProfile);
+
+    service.deleteProgrammeMembershipsForTrainee(TRAINEE_TIS_ID);
+
+    verify(cachingDelegate, times(1))
```

Comment:
nit-picking, but using `times(1)` is the same behaviour as `verify(cachingDelete).blah`

Reply:
Ta, removed 😁